### DISCUSSION TITLE
Spawning without weapons

### DISCUSSION THREAD

**Discussion by SunFlowerXiXi** [Joe91/fun-bots#277] [Q&A] [answered]
Occasionally my friends and I will spawn without any weapons and be unable to shoot or do anything. We can not pick up kits from downed players, but can still interact with vehicles. I see this issue described a lot on other forums but never resolved. This happens very often, to myself and all of my friends and we have not discovered a way to resolve the issue. Has anyone figured out why this happens and what we can do to resolve it?

**Comment by Digitalinsite** [comment]
I havent found a way to solve it specifically but it seems to happen more the higher the number of bots you have in your server. I changed my max bots per team to 24 or 25 and I rarely see it happen now if it all. I suspect the game engine is hitting an internally coded memory allocation limit.

**Comment by DefSw0rd** [comment, marked as answer]
Open %funbotModFolder%/ext/shared/Registry/Registry.lua, there is one option "USE_LOAD_BUNDLE_BUGFIX = false", try to change it to "true" (Ofcourse the dev team has claimed"These variables should not be configured by the end-user" 🤣 )
This might help. Anyway, in my 2.7.0 game experience, it helps a little (weapon still disappear sometimes)[Caspian Border CQ Large 28 bots per team]
